README.md

```diff
@@ -24,39 +23,60 @@ the output is of size as `(nrows, ncols_v)`. If all input tensors are on GPU, a
 
 
 ## Installation
+After setting up an environment with pytorch >= 1.3, run either of these commands from the root folder of the repo: 
 
-Just `python setup.py install`, in the root folder of this repo. This will compile
-and install the torchsearchsorted module.
-be careful that sometimes, `nvcc` needs versions of `gcc` and `g++` that are older than those found by default on the system. If so, just create symbolic links to the right versions in your cuda/bin folder (where `nvcc` is)
+```bash
+pip install -v .
+```
```

Comment:
Your modification removes the very important information that nvcc sometimes needs a version of gcc and g++ that are below the default ones on the system, so that your installation procedure will not work

Reply:
All right, sorry for removing that disclaimer. I tested on my machine and on a few servers we have here and did not notice any issue, that's why it felt redundant to me. Also, `g++` is usually just a link to `gcc`, I haven't seen cases where the two are different. 

Anyway, let's just restore that warning so that no one runs into problems ;)